src/histolab/tiler.py

```diff
@@ -28,8 +29,8 @@ class Tiler(Protocol):
     tile_size: int
 
     @lru_cache(maxsize=100)
-    def box_mask(self, slide: Slide) -> sparse._coo.core.COO:
-        """Return binary mask at level 0 of the box to consider for tiles extraction.
+    def box_mask_thumb(self, slide: Slide) -> np.ndarray:
```

Comment:
I'm not sure about the name of this method. It's not meaningful to me ... why did you change `box_mask` onto this new name?

Reply:
Before there were both `box_mask` (mask at level 0) and `box_mask_lvl` (mask at level of extraction) methods. Now there is only the method to return the mask at thumbnail level and I didn't want to keep the same method name and return a completely different thing... but if you think that we could keep only `box_mask` with a different meaning, I'm okay with that

Reply:
considering that now we have only 1 name i'd prefer `box_mask` but TBH it's up to you :D

Reply:
ok changed in #128